lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.js

```diff
@@ -120,7 +120,7 @@ module.exports = {
     if (apiGateway.minimumCompressionSize) {
       const minimumCompressionSize = apiGateway.minimumCompressionSize;
 
-      if (!_.isInteger(minimumCompressionSize)) {
+      if (!Number(minimumCompressionSize)) {
```

Comment:
We should remove this condition entirely (note that if value is not an integer, at this point it'll be neatly reported by validation on AWS CloudFormation side, and we also have shortly coming schema based validation https://github.com/serverless/serverless/issues/6562 that will further improve that experience)

Reply:
@medikoo There is a fail test case in ```restApi.test.js``` when I remove this condition entirely. Should I remove that test case?
This is the error:
> 4) #compileRestApi()
       should throw error if minimumCompressionSize is not an integer:
     AssertionError: expected promise to be rejected with 'Error' but it was fulfilled with undefined
      at Proxy.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:169:53)
      at Proxy.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:47:29)
      at Proxy.methodWrapper (node_modules\chai\lib\chai\utils\addMethod.js:57:25)
      at Context.it (lib\plugins\aws\package\compile\events\apiGateway\lib\restApi.test.js:223:64)
      at process._tickCallback (internal/process/next_tick.js:61:11)

Reply:
@nvdai2401 yes let's remove this test.